nextcord/types/interactions.py

```diff
@@ -228,6 +228,13 @@ class Interaction(_InteractionOptional):
     version: int
 
 
+class MessageInteraction(TypedDict):
+    id: Snowflake
+    type: InteractionType
+    name: str
+    user: User
```

Comment:
why was this just moved?

Reply:
it was not, the one below was a duplicate (very stupid mistake ik)